tests/timetables/test_interval_timetable.py

```diff
@@ -40,6 +40,31 @@
 HOURLY_TIMEDELTA_TIMETABLE = DeltaDataIntervalTimetable(datetime.timedelta(hours=1))
 HOURLY_RELATIVEDELTA_TIMETABLE = DeltaDataIntervalTimetable(dateutil.relativedelta.relativedelta(hours=1))
 
+CRON_TIMETABLE = CronDataIntervalTimetable("30 21 * * *", TIMEZONE)
+
+@pytest.mark.parametrize(
+    "timetable",
+    [
+        pytest.param(CRON_TIMETABLE, id="cron"),
+    ],
+)
+@pytest.mark.parametrize(
+    "last_automated_data_interval",
+    [pytest.param(None, id="first-run"), pytest.param(PREV_DATA_INTERVAL, id="subsequent")],
+)
+@freezegun.freeze_time(CURRENT_TIME)
+def test_no_catchup_first_starts_at_current_time(
+    timetable: Timetable,
+    last_automated_data_interval: Optional[DataInterval],
+) -> None:
+    """If ``catchup=False`` and start_date is a day before"""
+    next_info = timetable.next_dagrun_info(
+        last_automated_data_interval=last_automated_data_interval,
+        restriction=TimeRestriction(earliest=CURRENT_TIME-datetime.timedelta(days=1), latest=None, catchup=False),
```

Comment:
But can you add a vector to paramtrize `earliest`? One for this value you use here, another for something like `CURRENT_TIME - datetime.timedelta(days=1, seconds=2)`.

Reply:
I am new to this, thank you for your guidance